_episodes/08-singularity-mpi.md

```diff
@@ -190,9 +215,9 @@ We can now try running a 2-process MPI run of a point to point benchmark `osu_la
 >
 > It is now necessary to create a `Slurm` job submission script to run the benchmark example.
 >
-> Download this [template script]({{site.url}}{{site.baseurl}}/files/osu_latency.slurm.template) _on the cluster_ and edit it to suit your configuration. (Copy the link and use a tool such as `wget` to download the template)
+> Download this [template script]({{site.url}}{{site.baseurl}}/files/osu_latency.slurm.template) _on the cluster_ and edit it to suit your configuration. (_Copy the link and use a tool such as `wget` to download the template._)
```

Comment:
I think my only comment is that the job script for the version built with `docker build` on Arm-based systems will be slightly different as you will need to specify the full path the executable. We can just highlight this at the time (if anyone is using a recent Mac) or we can add some text.